### PR TITLE
Space Travelers Project

### PR DESCRIPTION
**Pull request by taldr27**
# Space Travelers Project
In this project we @J2ZROMERO @taldr27:

- Used React documentation.
- Used React components.
- Used API to fetch data from Space-X missions and rockets.
- Used React props.
- Used React Router.
- Connected React and Redux.
- Handled events in a React app.
- Used styles in a React app.
- Applied React best practices and language style guides in code.
- Used store, actions, and reducers in React.
- Performed a code review between team members.
- Wrote unit tests with React Testing Library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
-# space-travelers
\ No newline at end of file
+
+# Getting Started
+
+**[Space travelers]** You can see several pictures with a short description about missions and rockets and you can simulate a making reservation with one of them.
+
+# Built With
```
Comment (algerina) [inline README.md:6]: - [ ] Kindly, use the Microverse template for  your Readme file.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @taldr27 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- You have a descriptive PR and commits
- No Linter errors
- Your project looks awesome



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by J2ZROMERO** [comment]
- Use Microverse Template

File: README.md (hunk 1 above)
Comment (J2ZROMERO) [reply README.md]: thanks for your feedback @algerina

**Review by Ismailco — APPROVED** [review]
### Hi @taldr27, @algerina, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**